ElectricCar.hpp

```diff
@@ -5,12 +5,11 @@
 class ElectricCar : public virtual Car
 {
 protected:
-    void charge();
+    static void charge();
```

Comment:
Też pytanie czemu static? Chcemy ładować jakiś konkretny samochód, więc ta metoda powinna działać na obiekcie. Jak jest static to działa na klasie.